tox.ini

```diff
@@ -66,7 +66,6 @@ basepython = python3.10
 usedevelop = True
 deps =
     {[testenv]deps}
-    git+ssh://github.com/docker/docker-py@3f0095a7c1966c521652314e524ff362c24ff58c#egg=docker
```

Comment:
here and below we don't need to install docker as it is already included in `setup.py`